mappings/net/minecraft/world/gen/chunk/OverworldChunkGeneratorConfig.mapping

```diff
@@ -1 +1,4 @@
 CLASS btu net/minecraft/world/gen/chunk/OverworldChunkGeneratorConfig
+	METHOD v getBiomeSize ()I
+	METHOD w getRiverSize ()I
+	METHOD x isFixedBiome ()I
```

Comment:
This returns an int, not a boolean, yet its name starts with `is`.

Reply:
That's not true! If it's not negative, then its value is returned when the layer is sampled, instead of running the sample logic. It should be `getFixedBiome`!